documentation/en/cli-lotus-miner.md

```diff
@@ -10,65 +11,69 @@ VERSION:
-   --create-worker-key                                        create separate worker key (default: false)
-   --worker value, -w value                                   worker key to use (overrides --create-worker-key)
-   --owner value, -o value                                    owner key to use
-   --sector-size value                                        specify sector size to use
-   --pre-sealed-sectors value [ --pre-sealed-sectors value ]  specify set of presealed sectors for starting as a genesis miner
-   --pre-sealed-metadata value                                specify the metadata file for the presealed sectors
-   --nosync                                                   don't check full-node sync status (default: false)
-   --symlink-imported-sectors                                 attempt to symlink to presealed sectors instead of copying them into place (default: false)
-   --no-local-storage                                         don't use storageminer repo for sector storage (default: false)
-   --gas-premium value                                        set gas premium for initialization messages in AttoFIL (default: "0")
-   --from value                                               select which address to send actor creation message from
-   --confidence value                                         number of block confirmations to wait for (default: 5)
-   --help, -h                                                 show help
+   --actor                        specify the address of an already created miner actor (default: "")
+   --genesis-miner                enable genesis mining (DON'T USE ON BOOTSTRAPPED NETWORK) (default: "false")
+   --create-worker-key            create separate worker key (default: "false")
+   --worker, -w                   worker key to use (overrides --create-worker-key) (default: "")
+   --owner, -o                    owner key to use (default: "")
+   --sector-size                  specify sector size to use (default: "")
+   --pre-sealed-sectors           --pre-sealed-sectors value [ --pre-sealed-sectors value ]	specify set of presealed sectors for starting as a genesis miner
+   --pre-sealed-metadata          specify the metadata file for the presealed sectors (default: "")
+   --nosync                       don't check full-node sync status (default: "false")
+   --symlink-imported-sectors     attempt to symlink to presealed sectors instead of copying them into place (default: "false")
+   --no-local-storage             don't use storageminer repo for sector storage (default: "false")
+   --gas-premium                  set gas premium for initialization messages in AttoFIL (default: "0")
+   --from                         select which address to send actor creation message from (default: "")
+   --confidence                   --confidence value	number of block confirmations to wait for (default: 0)
+   --help, -h                     show help (default: "false")
+
+COMMANDS:
```

Comment:
4. `COMMANDS:` is below the `OPTIONS:` here

Reply:
Fixed.